Shoko.Server/API/v3/Controllers/ImageController.cs

```diff
@@ -20,64 +21,38 @@ public class ImageController : BaseController
     /// <summary>
     /// Returns the image for the given <paramref name="source"/>, <paramref name="type"/> and <paramref name="value"/>.
     /// </summary>
-    /// <param name="source">AniDB, TvDB, MovieDB, Shoko</param>
-    /// <param name="type">Poster, Fanart, Banner, Thumb, Static</param>
-    /// <param name="value">Usually the ID, but the resource name in the case of image/Shoko/Static/{value}</param>
+    /// <param name="source">AniDB, TvDB, TMDB, Shoko, etc.</param>
+    /// <param name="type">Poster, Fanart, Banner, Thumbnail, etc.</param>
+    /// <param name="value">The image ID.</param>
     /// <returns>200 on found, 400/404 if the type or source are invalid, and 404 if the id is not found</returns>
     [HttpGet("{source}/{type}/{value}")]
     [ProducesResponseType(typeof(FileStreamResult), 200)]
     [ProducesResponseType(404)]
     public ActionResult GetImage([FromRoute] Image.ImageSource source, [FromRoute] Image.ImageType type,
-        [FromRoute] string value)
+        [FromRoute] int value)
```

Comment:
This looks like a breaking change. Please comment

Reply:
Also, the way we use the image metadata in the web ui makes it so this change won't break the web ui. Shokofin might break, but I can fix that in the "unstable" channel if it does break it.

Reply:
Looking at @harshithmohan's v3 branch for Metadata, then it doesn't look like it will break since it's just using the JSON values as is without any type checking (which is done in Shokofin, since it's C#).

https://github.com/harshithmohan/ShokoMetadata.bundle/blob/apiv3/Contents/Code/__init__.py#L481